### ISSUE TITLE
Twitter feed will not load when script is enabled

### ISSUE THREAD

**Issue by Larkstarr** [59de44955ebd/twitter-to-bsky#8]
Firefox 126.0.1
Tampermonkey 5.1.1
Script 0.14

As it says on the tin, if the script is enabled in Tampermonkey, Twitter displays "something went wrong" and won't load. If I disable the script, all becomes well.

Works fine in Chrome.

If I can forward any logs or anything to help please let me know!

**Comment by 59de44955ebd** [comment]
Thanks, unfortunately I can confirm it. It's not about the current Firefox/Tampermonkey/Script version, but Twitter/X itself must have changed something in its code that currently makes the script dysfunctional in Firefox. I'll try to find out what exactly causes the problem, but to be honest, at the moment I'm clueless.

**Comment by 59de44955ebd** [comment]
Update: it seems that Tampermonkey's content.js is now somehow constantly blocked by Twitter/X in Firefox (but I still don't know how). But when deactivating Tampermonkey and instead activating Violentmonkey, twitter-to-bsky works just like before, it loads fine about every other load of the Twitter page, and usually always loads fine when the Twitter/X page is loaded via a shift-reload.
So it **might** be the case that the Twitter guys found a way to explicitely block Tampermonkey in Firefox - but that's just an assumption.

**Comment by 59de44955ebd** [comment]
Nope, that was a stupid conspiracy theory ;-) Instead, it seems that Tampermonkey in Firefox now behaves exactly as Violentmonkey already did before: sometimes the script loads fine in Twitter/X, sometimes it does not. And usually it loads fine when the Twitter/X page was shift reloaded.

As it seems so random, I honestly don't know how to deal with this. Other than suggesting to use Chrome instead (but I'm a Firefox user myself and hate Chrome).

**Comment by kawaiikaorichan** [comment]
So you would recommend using Violentmonkey instead of Tampermonkey in Firefox? Is frustating using Ctrl + F5 each time I open a new Twitter tab.

**Comment by valdisvi** [comment]
I can confirm this issue as well when using Tampermonkey.

**Comment by ghost** [comment]
Neither ViolentMonkey or Tampermonkey work. Tampermonkey has the loading issue, and ViolentMonkey cant inject the script. Both only issues on firefox apparently cuz it works on Chromium.

**Comment by valdisvi** [comment]
I use 64-bit Firefox 128.4.0esr on Linux and it works besides this issue.
